class InputError(Exception):
    """Исключение для ошибок ввода пользователя."""
    pass


class BookError(Exception):
    """Исключение для ошибок, связанных с книгами."""
    pass